### DISCUSSION TITLE
Install LTSP in a lxc-container

### DISCUSSION THREAD

**Discussion by sneggyra** [ltsp/ltsp#191] [Q&A]
Is it possible to install the ltsp-server in a lxc-container in proxmox or do I have to create a "full" VM?

**Comment by alkisg** [comment]
I haven't used lxc containers, it depends on whether they behave nice with loop devices, (bind) mounts, starting services in <1024 ports like nfs etc.

E.g. a quick googling shows that nfs might need [this](https://gist.github.com/rwenz3l/0907385f6a6690c34eb8e36fa73d8405).

Maybe you could just start testing it and ping if you see any issues...

**Comment by sneggyra** [comment]
I tested to run ltsp-server in a proxmox-lxc-container.
Options nfs and cifs are enabled for this container in proxmox.
I get an error by running "ltsp image /" (see attached image)

It seems that I have to load the "overlay" kernel-module in the container.
That seems to me that is a little bit difficult and I dont't now if this is the complete solution.

For reasons of time I use a full VM now. That works fine.

![Bildschirmfoto vom 2019-10-21 11-52-07](https://user-images.githubusercontent.com/56351321/67195625-90028000-f3f9-11e9-96bc-3dab6d6a81ca.png)

**Reply by panticz** [reply]
Try to load the overlay kernel module on the hypervisor:

`modprobe overlay`
